src/puppetlabs/trapperkeeper/services_internal.clj

```diff
@@ -58,6 +58,13 @@
    :dependencies (schema/pred sequential?)
    :fns-map FnsMap})
 
+(def ProtocolandDependenciesMap
+  "Schema defining the map used to represent the protocol and dependencies
+  of a service."
+  {:fns (schema/pred seq?)
```

Comment:
Technically, this could be a little more strict.  We know that, at least, they should be a seq of seqs, so you could maybe do `(schema/pred #(and (seq? %) (every? seq? %)))`.  Since `every?` would probably fail on anything that isn't a seq/coll, it might be ok to just do `(schema/pred #(every? seq? %))`.

You could try to get even more advanced inside of there, since we kind of know more about the specific forms that are necessary for a seq to represent a fn def, like `(symbol? (first %))`, and `(vec? (second %))` (i.e., to match a fn definition like `(foo [arg1 arg2] ...)`, but I'm not sure that's necessary (we weren't doing it before).